.github/workflows/release.yml

```diff
@@ -49,6 +51,8 @@ jobs:
     steps:
       - name: Checkout cli
         uses: actions/checkout@v2
+        with:
+          persist-credentials: false
```

Comment:
Is this to stop persisting the auth tokens? Is it by-default set to `true`? If yes, don't we need to have this at all places?

Reply:
Okay. 
I meant do we need to add this `persist-credentials: false` everywhere else for security reasons, maybe in a cleanup stage? Just trying to ensure that the token is not valid after the job is done.

Reply:
As per the [docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication), **Before each job begins, GitHub fetches a token for the job. The token expires when the job is finished.**.